.github/workflows/deploy-grafana-dashboards.yaml

```
@@ -35,6 +35,8 @@ jobs:
 
       - name: Setup sops
         uses: mdgreenwald/mozilla-sops-action@v1
+        with:
+          version: v3.7.2
```

I believe `3.7.1` is latest working version, right?

```suggestion
          version: v3.7.1
```